### PR TITLE
Use appveyor docker support

### PR DESCRIPTION
**Pull request by friism**
To get this hooked up you have to request appveyor access to WS2016 here: https://github.com/appveyor/ci/issues/1150

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-branches:
-  only:
-    - master
-    - release
-    - dev
-    - /^(.*\/)?ci-.*$/
+image: Windows Server 2016
+
+install:
+  - ps: Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.11.2/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
```
Comment (StefanScherer) [inline appveyor.yml]: You may use `choco install docker-compose` as AppVeyor also has Chocolatey installed ;-)

Comment (friism) [inline appveyor.yml]: This is simpler because I won't have to also install chocolatey

Comment (StefanScherer) [inline appveyor.yml]: I thought the same, but later I realized it is already installed and listed in https://www.appveyor.com/docs/installed-software/#tools

File: appveyor.yml [hunk 2]
```diff
@@ -1,14 +1,13 @@
-init:
-  - git config --global core.autocrlf true
-branches:
-  only:
-    - master
-    - release
-    - dev
-    - /^(.*\/)?ci-.*$/
+image: Windows Server 2016
```
Comment (StefanScherer) [inline appveyor.yml]: Should be updated to `Visual Studio 2017` to have official Docker build in AppVeyor.

File: appveyor.yml [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
+
+install:
+  - ps: Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.11.2/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
```
Comment (StefanScherer) [inline appveyor.yml]: This can be removed as docker-compose is already installed in AppVeyor.

File: appveyor.yml [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
+  - ps: Set-MpPreference -DisableRealtimeMonitoring $true
+
 build_script:
+  - docker-compose -f .\docker-compose.windows.yml up -d
```
Comment (StefanScherer) [inline appveyor.yml]: Yes, that would be a great test if the Docker build really works on Windows. Just wondered after looking at the Dockerfile.windows and realized that it currently uses a Linux base image.

**Comment by natemcmaster** [comment]
Thanks for the contribution. Sorry it fell off our radar. At this point, this PR is stale and won't work anymore as this repo has moved on to .NET Core 2.0.

**Pull request closed without merging**